tests/api/test_statements_post.py

```diff
@@ -102,6 +102,71 @@ async def test_api_statements_post_single_statement_directly(  # noqa: PLR0913
     )
 
 
+@pytest.mark.anyio
+@pytest.mark.parametrize(
+    "backend",
+    [
+        get_async_es_test_backend,
+        get_async_mongo_test_backend,
+        get_es_test_backend,
+        get_clickhouse_test_backend,
+        get_mongo_test_backend,
```

Comment:
Missing `get_fs_test_backend`?

Reply:
Yes, it seems the `fs` backend is not tested for a POST. Will need to add it in another PR